packages/astro/src/@types/astro.ts

```diff
@@ -1527,7 +1578,7 @@ export interface SSRLoadedRenderer extends AstroRenderer {
 	};
 }
 
-export type HookParameters<
```

Comment:
This rename would be a breaking change. Can you add this back as an alias (maybe with a TODO to remove in 3.0)?

```ts
export type HookParameters = HookIntegrationParameters;
```

Reply:
Good catch! I will revert the change, it's an old code